content/05.methods.md

```diff
@@ -176,7 +195,7 @@ We also explored the performance of vg and SMRT-SV2 in different sets of regions
 ### Yeast graph analysis
 
 For the analysis of graphs from de novo assemblies, we utilized publicly available PacBio-derived assemblies and Illumina short read sequencing datasets for 12 yeast strains from two related clades (Table {@tbl:strains}) [@doi:10.1038/ng.3847].
-Five strains were selected (two from different subclades of each clade plus the reference *S.c. S288C*): *S.c. SK1*, *S.c. YPS128*, *S.p. CBS432*, *S.p. UFRJ50816*, and *S.c. S288C*.
+Five strains were selected for contructing a graph (two from different subclades of each clade plus the reference *S.c. S288C*): *S.c. SK1*, *S.c. YPS128*, *S.p. CBS432*, *S.p. UFRJ50816*, and *S.c. S288C*.
```

Comment:
Maybe "for graph construction" would be better because we don't construct "a graph" but rather two graphs?!